content/W2017/W17-event-beckwith-talk.md

```diff
@@ -0,0 +1,29 @@
+Title: Tech Talk: Hotspot Virtual Machine's Execution Engine by Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: events, academic
+Slug: beckwith-tech-talk-w17
+Author: Monica Xu
+Summary: Come listen about the amazing work Monica Beckwith is doing!
+
+Monica Beckwith has worked with the Java Virtual Machine 
```

Comment:
can you add an intro before this paragraph that looks like the first paragraph in http://wics.uwaterloo.ca/2016/02/stephanie-morillo-talk/ ?